js-pkg/apps/demos/src/pages/webrtc-demo.jsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react'
+import { DriftDBProvider, usePresence, useLatency } from 'driftdb-react'
+import { Chat } from '../components/chat.jsx'
+import { DRIFTDB_URL } from '../config'
+
+const Peers = ({ peers, myLatency }) => {
+  const listItems = peers.map(([sessionPeer, { id, latency }]) => (
+    <tr key={id}>
+      <td className="border px-4 mx-3"> {sessionPeer} </td>
+      <td className="border px-4 mx-3"> {id} </td>
+      <td className="border px-4 mx-3"> {latency + myLatency} ms </td>
```

Comment:
Let's format this so we don't have `75.99999999999999999999998 ms` in the table.

Something like this maybe?:

```js
function formatFloat(num, decimals) {
  const mult = Math.pow(10, decimals)
  return Math.round(num * mult) / mult
}
```